SDKAndroid/src/main/java/app/cybrid/sdkandroid/util/Utils.kt

```diff
@@ -27,4 +28,10 @@ fun getDateInFormat(date: OffsetDateTime, pattern:String = "MMM dd, YYYY"): Stri
 
     val formatter = DateTimeFormatter.ofPattern(pattern)
     return date.format(formatter)
+}
+
+fun getImageUrl(name: String): String {
+
+    val cybrid = Cybrid.instance
+    return "${cybrid.imagesUrl}$name${cybrid.imagesSize}"
```

Comment:
What happens if this URL is not available?

Reply:
I set a placeholder image im gray background.